src/schema/v2/artwork/index.ts

```diff
@@ -334,6 +335,30 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return normalizeImageData(size ? _.take(sorted, size) : sorted)
         },
       },
+      inquiryQuestions: {
+        type: new GraphQLList(InquiryQuestionType),
```

Comment:
Do you think it's worth adding a comment that this is fixture/placeholder data for now? Outside of the context of this PR it's not super obvious.

Reply:
Will do